format.sh

```diff
@@ -1,3 +1,4 @@
 #! /bin/bash
 
 git ls-files -- src*.cpp src*.hpp testsuite*.cpp testsuite*.hpp | xargs clang-format -i -style=file
+git ls-files -- src*.py testsuite*.py | xargs autopep8 --in-place --aggressive --aggressive 
```

Comment:
Maybe also include `examples*.py`, or should it be done after #369?

Reply:
I think both are independent. I will include examples.